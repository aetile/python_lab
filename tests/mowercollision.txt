3 3
1 2 N
DA
2 0 N
AAAA
