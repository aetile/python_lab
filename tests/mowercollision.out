2 2 E
2 1 N
